package.json

```diff
@@ -131,8 +131,9 @@
     "eslint-plugin-storybook": "^0.6.15",
     "husky": "^7.0.2",
     "ignore": "^5.2.4",
-    "jest": "^27.4.3",
+    "jest": "^28.1.1",
     "jest-axe": "^6.0.0",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
here and in all other package.json files, we have to explicitly install `jest-environment-jsdom` due to the jest 28 upgrade: https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28#jsdom for a short note; https://jestjs.io/blog/2021/05/25/jest-27#flipping-defaults for longer context